routes/order.routes.js

```diff
@@ -0,0 +1,36 @@
+const orderController=require("../controllers/order.controllers")
+const {authValidation,ordervalid}=require("../middelware/index")
+module.exports=(app)=>{
+    /*
+                (createOrder)
+         POST   /orderService/api/v1/orders
+    */
+    app.post("/orderService/api/v1/orders",
+            [ordervalid.orderValidation,authValidation.verifyToken],
```

Comment:
verifyToken should happen first... Always think about the order of MW when chaining it